main.go

```diff
@@ -18,57 +18,60 @@ package main
 
 import (
 	"flag"
-	"log"
+	"fmt"
 	"os"
 
 	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
 	// to ensure that exec-entrypoint and run can make use of them.
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 
 	"github.com/civo/civogo"
-	infrastructurev1alpha1 "github.com/null-channel/cluster-api-provider-civo/api/v1alpha1"
-	"github.com/null-channel/cluster-api-provider-civo/controllers"
 	"k8s.io/apimachinery/pkg/runtime"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/healthz"
-	"sigs.k8s.io/controller-runtime/pkg/internal/log"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	infrastructurev1alpha1 "github.com/null-channel/cluster-api-provider-civo/api/v1alpha1"
+	infrastructurev1beta1 "github.com/null-channel/cluster-api-provider-civo/api/v1beta1"
+	"github.com/null-channel/cluster-api-provider-civo/controllers"
 	//+kubebuilder:scaffold:imports
 )
 
+const (
+	defaultRegion = "London 1"
```

Comment:
regions should use short codes so "London 1" should be "LON1"

Reply:
Fixed

Reply:
nice catch!